Bypass all tests since all are false positives on Windows or Solaris
diff --git a/test/ruby/test_time_tz.rb b/test/ruby/test_time_tz.rb
index 43758d948f..26c11b1a53 100644
--- a/test/ruby/test_time_tz.rb
+++ b/test/ruby/test_time_tz.rb
@@ -1,4 +1,5 @@
 # frozen_string_literal: false
+# class not created on mingw, mswin, or solaris
 require 'test/unit'
 require '-test-/time'
 
@@ -476,4 +477,4 @@ def self.gen_variational_zdump_test(hint, data)
 Europe/Lisbon  Mon Jan  1 00:36:44 1912 UT = Sun Dec 31 23:59:59 1911 LMT isdst=0 gmtoff=-2205
 Europe/Lisbon  Sun Dec 31 23:59:59 1911 UT = Sun Dec 31 23:23:14 1911 LMT isdst=0 gmtoff=-2205
 End
-end
+end unless RUBY_PLATFORM[/mingw|mswin|solaris/]
